### PR TITLE
[ETFE-3062] - Added transport arranger review answers page (COD-06)

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

File: app/controllers/sections/transportArranger/TransportArrangerIndexController.scala [hunk 1]
```diff
@@ -40,7 +40,7 @@ class TransportArrangerIndexController @Inject()(
   def onPageLoad(ern: String, arc: String): Action[AnyContent] =
     authorisedDataRequestWithUpToDateMovement(ern, arc) { implicit request =>
       if (TransportArrangerSection.isCompleted || TransportArrangerSection.needsReview) {
```
Comment (james-work-account) [inline app/controllers/sections/transportArranger/TransportArrangerIndexController.scala:42]: We might want to change this (and other index routes) to be something more like this:

```scala
if(Section.isCompleted) {
  // CYA route
} else if(Section.needsReview) {
  // review route
} else {
  // section start
}
```

File: app/navigation/TransportArrangerNavigator.scala [hunk 2]
```diff
@@ -26,6 +27,7 @@ import play.api.mvc.Call
 
 import javax.inject.Inject
 
+//noinspection ScalaStyle
```
Comment (james-work-account) [inline app/navigation/TransportArrangerNavigator.scala:30]: why?

File: app/controllers/sections/transportArranger/TransportArrangerIndexController.scala (hunk 1 above)
Comment (LewisMitchell) [reply app/controllers/sections/transportArranger/TransportArrangerIndexController.scala:42]: Matt was going to deal with the .isCompleted stuff and going to CYA, so hence why I left it. He said it was easier to do it in his

File: app/navigation/TransportArrangerNavigator.scala (hunk 2 above)
Comment (LewisMitchell) [reply app/navigation/TransportArrangerNavigator.scala:30]: Cyclomatic complexity was more than 10, in other navigators this is present. I could split the routing logic into a private method but wanted to keep it consistent with other navs. Up to you?

Comment (james-work-account) [reply app/navigation/TransportArrangerNavigator.scala:30]: Nah that's fine, just wanted to double check it wasn't a temporary thing that slipped in 🙂

File: app/controllers/sections/transportArranger/TransportArrangerIndexController.scala (hunk 1 above)
Comment (MJCallahanPage) [reply app/controllers/sections/transportArranger/TransportArrangerIndexController.scala:42]: Yeah, already on it.

**Review by james-work-account — APPROVED** [review]

**Pull request merged**